tests/integration/inventories/eos.yaml

```diff
@@ -0,0 +1,19 @@
+---
+all:
+  hosts:
+    localhost:
+      ansible_connection: local
+      ansible_python_interpreter: /bin/python3.8
+appliance:
+  hosts:
+    eos:
+      ansible_host: eos.example.org
+      ansible_user: zuul
+  vars:
+    ansible_connection: ansible.netcommon.network_cli
```

Comment:
we likely need to set: ansible_become_method: ansible.netcommon.enable

https://github.com/ansible/network-ee/pull/104/files#diff-495d8d00fb404ca495981e250bd6d68a598a15c6b53ceea0fd9265b8ee9b7946R14